_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -0,0 +1,235 @@
+
+We support migrating a document-replication based cluster to Remote-backed storage through Rolling Upgrade mechanism.
+
+Rolling upgrades, sometimes referred to as "node replacement upgrades", can be performed on running clusters with virtually no downtime. Nodes are individually stopped and upgraded in place. Alternatively, nodes can be stopped and replaced, one at a time, by hosts running the new version. During this process you can continue to index and query data in your cluster.
+
+## Preparing to migrate
+
+Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-opensearch/index/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indices before you make any changes to your OpenSearch cluster.
+
+Users need to move to OpenSearch 2.14 version as a pre-requisite of this migration.
+
+**Important:** OpenSearch nodes cannot be migrated back to document replication as of 2.14. If you need to revert the migration, then you will need to perform a fresh installation of OpenSearch and restore the cluster from a snapshot. Take a snapshot and store it in a remote repository before beginning the upgrade procedure.
+{: .important}
+
+## Performing the upgrade
+
+1. Verify the health of your OpenSearch cluster before you begin. You should resolve any index or shard allocation issues prior to upgrading to ensure that your data is preserved. A status of **green** indicates that all primary and replica shards are allocated. See [Cluster health]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-health/) for more information. The following command queries the `_cluster/health` API endpoint:
+   ```json
+   GET "/_cluster/health?pretty"
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+       "cluster_name":"opensearch-dev-cluster",
+       "status":"green",
+       "timed_out":false,
+       "number_of_nodes":4,
+       "number_of_data_nodes":4,
+       "active_primary_shards":1,
+       "active_shards":4,
+       "relocating_shards":0,
+       "initializing_shards":0,
+       "unassigned_shards":0,
+       "delayed_unassigned_shards":0,
+       "number_of_pending_tasks":0,
+       "number_of_in_flight_fetch":0,
+       "task_max_waiting_in_queue_millis":0,
+       "active_shards_percent_as_number":100.0
+   }
+   ```
+1. Disable shard replication to prevent shard replicas from being created while nodes are being taken offline. This stops the movement of Lucene index segments on nodes in your cluster. You can disable shard replication by querying the `_cluster/settings` API endpoint:
```

Comment:
```suggestion
1. Disable shard replication to prevent shard replicas from being created while nodes are being taken offline. This stops the movement of Lucene index segments on nodes in your cluster. You can disable shard replication by querying the `_cluster/settings` API endpoint, as shown in the following example:

```